### ISSUE TITLE
no callback function warning

### ISSUE THREAD

**Issue by FredSoderberg** [ErrorPro/react-google-autocomplete#200]
`Loading the Google Maps JavaScript API without a callback is not supported: https://developers.google.com/maps/documentation/javascript/url-params#required_parameters`

We are receiving this warning when we are using the `usePlacesService`. This warning seems to have popped up just recently for us because we haven't seen it during implementation or in staging environments and now it appears everywhere.

**Comment by ErrorPro** [comment]
@FredSoderberg can y try to use @2.7.1 version? If that doesn't help then we would probably have to look into the google's lib

**Comment by patoaesteve** [comment]
Hi! I was using version 2.4.0 until yesterday without any problems (<Autocomplete apiKey={... />). Today i'm having the same error mentioned (Loading the Google Maps JavaScript API without a callback is not supported: https://developers.google.com/maps/documentation/javascript/url-params#required_parameters) . Same with latest version

**Comment by solarpush** [comment]
Hi, a solution for this err ? 
-v 2.7.1 || 2.6.1 same error : Loading the Google Maps JavaScript API without a callback is not supported:

**Comment by ananth99** [comment]
can confirm that we're also facing the same issue for `2.6.0` since the last 3 days

**Comment by FredSoderberg** [comment]
> @FredSoderberg can y try to use @2.7.1 version? If that doesn't help then we would probably have to look into the google's lib

We tested 2.7.1 and 2.7.2 but to no avail. We will switch to manually loading the api just to be sure that functionality is preserved over time. I can also mention that even though we receive this warning we cant detect any other problems.

**Comment by violy** [comment]
Same issue on `2.7.2`. I've patch this with this code

```ts
window && (window._gmapsapicb = () => {});
const {
  // ...
} = usePlacesService({
  apiKey: MY_API_KEY + "&callback=_gmapsapicb",
});
```

where `_gmapsapicb` is an arbitrary noop callback function name…

**Comment by jrcairns** [comment]
Same issue on `2.6.1`. This stops the console errors for my project

```
  const { ref } = usePlacesWidget({
    apiKey: 'xxxxxxxxxxxxxxxxxxxx&callback=Function.prototype',
    ....
  });
```

**Comment by surefiresolutions** [comment]
1. It appears as of 1/21/23 (I think I read that) Google throws errors if no callback is included in the parameters when loading the Google Maps API script
2. A workaround is to omit the apiKey parameter and include the script call (with your API Key and callback function) manually
3. It would be great if the next package update includes a default callback being passed with an optional parameter for including a custom callback function

For now I think I am ignoring it. I am hopeful the package will be updated soon. Please correct me if I am wrong about any of this. Thank you!

**Comment by gmwill934** [comment]
Same issue here, but functionality is not broken

**Comment by ErrorPro** [comment]
Merged the fix provided by @arcdev1 in @2.7.3, please give it a try
